docs/developer-docs/defi/rosetta/icp_rosetta/index.md

```diff
@@ -0,0 +1,146 @@
+
+The Docker daemon (`dockerd`) should automatically start when you reboot your computer. If you start the Docker daemon manually, the instructions vary depending on the local operating system.
+
+### Step 2:  Pull the latest `dfinity/rosetta-api` image from the Docker Hub by running the following command:
+
+``` bash
+docker pull dfinity/rosetta-api
+```
+
+### Step 3:  Start the integration software by running the following command:
+
+``` bash
+docker run \
+    --interactive \
+    --tty \
+    --publish 8081:8081 \
+    --rm \
+    dfinity/rosetta-api
+```
+
+This command starts the software on the local host and displays output similar to the following:
+
+    Listening on 0.0.0.0:8081
+    Starting Rosetta API server
+
+By default, the software connects to a testnet.
+It **does not** connect to the ledger canister running on the Internet Computer blockchain mainnet.
+
+If you have been assigned a test network and corresponding ledger canister identifier, you can run the command against that network by specifying an additional `canister` argument.
+For example, the following command illustrates connecting to the ledger canister on a test network by setting the `canister` argument to `2xh5f-viaaa-aaaab-aae3q-cai`.
+
+``` bash
+docker run \
+    --interactive \
+    --tty \
+    --publish 8080:8080 \
+    --rm \
+    dfinity/rosetta-api \
+    --canister 2xh5f-viaaa-aaaab-aae3q-cai
+```
+
+:::info
+The first time you run the command, it might take some time for the node to catch up to the current link of the chain.
+When the node is caught up, you should see output similar to the following:
+
+You are all caught up to block height 109
+:::
+
+After completing this step, the node continues to run as a **passive** node that does not participate in block making.
+
+### Step 4:  Open a new terminal window or tab and run the `ps` command to verify the status of the service.
+
+If you need to stop the service, press <kbd>Ctrl-C</kbd>. You might want to do this to change the canister identifier you are using, for example.
+
+To test the integration after setting up the node, you will need to write a program to simulate a principal submitting a transaction or looking up an account balance.
+
+### Run the Rosetta node in production
+
+When you are finished testing, you should run the Docker image in production mode without the `--interactive`, `--tty`, and `--rm` command-line options.
+These command-line options are used to attach an interactive terminal session and remove the container, and are primarily intended for testing purposes.
+To run the software in a production environment, you can start the Docker image using the `--detach` option to run the container in the background and, optionally, specify the `--volume` command for storing blocks.
+
+To connect the Rosetta node instance to the mainnet, add flags: `--mainnet` and `--not-whitelisted`.
```

Comment:
The `--mainnet` flag is pretty self-explanatory, but maybe it would make sense to briefly explain what `--not-whitelisted` means, and why is needs to be specified? This is the first occurrence I see of it, but maybe it is explained somewhere else.